### ISSUE TITLE
Q2 2023 priorities

### ISSUE THREAD

**Issue by StephanTLavavej** [microsoft/STL#3628]
In the second quarter of 2023, our priorities are:

* Implementing [C++23 features](https://github.com/microsoft/STL/projects/8?fullscreen=true) from the Working Draft
  + C++23 was finalized in Feb 2023, so the remaining features are now monotonically non-increasing :chart_with_downwards_trend:
  + Ideally, we'll complete all C++23 library-only features in calendar year 2023! :tada:
* Keeping PR reviews flowing, so contributions are reviewed promptly
* Transcribing the [video code review backlog](https://github.com/microsoft/STL/wiki/Code-Review-Videos)
  + To make videos public, they need to be accessible, so correct captions are necessary. See https://github.com/strega-nil-ms/stl-video-transcripts for how you can help!

If you have questions about our plans for this quarter, feel free to ask them here. (Note that compiler, IDE, etc. priorities are off-topic.)

**Comment by jovibor** [comment]
Maybe there is some tiny whispers in barely hearing private talks about some pre-pre-preparations of vNext advent?

**Comment by StephanTLavavej** [comment]
Nope, sorry. I wish we were gearing up for vNext too (as the only STL maintainer who was around for the ABI break era).

**Comment by StephanTLavavej** [comment]
Good question - Mahmoud (my boss) has been working to get approval, with yet another meeting happening last week, but more work is needed. No ETA yet.

**Comment by lukehb** [comment]
Also wanting to throw my eagerness behind the whole

> open sourcing vcruntime and vcstartup?

Curious if internal approval has been given yet?

**Comment by StephanTLavavej** [comment]
Not yet - internal approval is the last thing we're trying to get. (Then it'll just be a matter of semi-mechanically cleaning up the sources and updating the build systems.)

**Comment by iainnicol** [comment]
> Not yet - internal approval is the last thing we're trying to get. (Then it'll just be a matter of semi-mechanically cleaning up the sources and updating the build systems.)

I appreciate you can't give ETAs, and there's only so much detail you can give... but that's progress isn't it? IIRC for a while there were technical blockers (performance concerns/benchmarking infrastructure). Sounds like good progress must have been made on that, and so the remaining approval is essentially a high up policy/legal process?

**Comment by StephanTLavavej** [comment]
Yes, we've made some progress. We've got benchmarking infrastructure in place, so we believe we have a response to any performance concerns. We also believe the path is clear for relicensing (this was a major effort). We just need final business approval, i.e. the ultrabosses giving us the green light. We just can't guarantee if or when that'll happen (we hope it will, but we are currently in a holding pattern).

We'll provide further updates when there are any to share.
